### PR TITLE
LevelOrderSuccessor/Solution

### PR DESCRIPTION
**Pull request by hpuma**

### CODE REVIEW

**Review by knrodriguez — CHANGES_REQUESTED** [review]
The file you are changing is inside the main assignments folder. Please copy and paste its contents to your personal folder, and revert this file to its original code. Thank you.

File: assignments/treesBFS/levelOrderSuccessor/levelOrderSuccessor.js [hunk 1]
```diff
@@ -49,4 +49,3 @@ if (result != null)
 result = find_successor(root, 9)
 if (result != null)
   console.log(result.val)
-
```
Comment (hpuma) [inline assignments/treesBFS/levelOrderSuccessor/levelOrderSuccessor.js:52]: Not sure why this is still lingering around, but it looks like a non interfering change

Comment (knrodriguez) [reply assignments/treesBFS/levelOrderSuccessor/levelOrderSuccessor.js:52]: The issue is that the file being edited is in this path `assignments/treesBFS/levelOrderSuccessor/levelOrderSuccessor.js`. The levelOrderSuccessor file should be added/edited inside your personal folder `henryPuma/assignments...`.

Comment (knrodriguez) [reply assignments/treesBFS/levelOrderSuccessor/levelOrderSuccessor.js:52]: Excuse my previous comment, if possible can you try and add that line back in? Let me know if it does not work.

**Review by knrodriguez — CHANGES_REQUESTED** [review]
If possible, try to add a line back in after line 51. Let me know if that does not work.

**Comment by hpuma** [comment]
I haven't been able to fix this, I'll close this PR and open a new one

**Pull request closed without merging**